src/wovn_interceptor.php

```diff
@@ -31,9 +33,13 @@
 $headers->requestOut();
 
 $uri = $headers->getDocumentURI();
-if (!Utils::isFilePathURI($uri, $store) &&
-    !Utils::isIgnoredPath($uri, $store)
-    ) {
+if (!Utils::isIgnoredPath($uri, $store)) {
+    $filePath = wovn_helper_detect_paths(__DIR__ . '/../..', $uri);
```

Comment:
is this trying to find the file on disk that is coming from the URL, so we can detect the mime type?

Reply:
Yep. I reused the same logic from wovn helper for that.

Reply:
Since we no longer check the request file, but only the response buffer, this is no longer a problem.